code/obj/critter/drone.dm

```diff
@@ -896,7 +903,7 @@
 	droploot = /obj/item/device/key/iridium
 	alertsound1 = 'sound/machines/engine_alert2.ogg'
 	alertsound2 = 'sound/machines/engine_alert3.ogg'
-	mats = 160
+	mats = list("POW-3" = 20, "iridiumalloy" = 30, "CON-2" = 21, "DEN-3" =22)
```

Comment:
Hm. I think that it would make sense for the Y-drone to not be scannable at all, seeing as it uses artifact tech etc

Reply:
sure, why not